src/index.js

```diff
@@ -1,32 +1,110 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import {BrowserRouter, Route, Switch} from 'react-router-dom';
+import {BrowserRouter, Link, Route, Switch} from 'react-router-dom';
 import {Provider} from 'react-redux';
 import store from './redux/store/index';
 
+import {
+  ACCOUNT,
+  ADD_BOOK,
+  ADMIN,
+  BOOKS,
+  BOOK_DETAIL,
+  LANDING,
+  MY_BOOK_HISTORY,
+  LOG_IN,
+  PASSWORD_FORGET,
+  SIGN_UP
+} from './routes';
+
 import Form from './components/Form/Form';
 import List from './components/List/List';
 import Post from './components/Post/Post';
-import Loader from './components/Loader/Loader';
+import Navbar from './components/Navbar/Navbar';
 
 import './styling/style.scss';
 
-const App = () => (
-  <React.Fragment>
-    <div>
-      <h2>Books</h2>
-      <List />
-    </div>
-    <div>
-      <h2>Add a new article</h2>
-      <Form />
-    </div>
-    <div>
-      <h2>API posts</h2>
-      <Post />
-    </div>
-  </React.Fragment>
-);
+const TestComponent = props => {
+  return (
+    <React.Fragment>
+      <div>
+        <h2>Books</h2>
+        <List />
+      </div>
+      <div>
+        <h2>Add a new book</h2>
+        <Form />
+      </div>
+      <div>
+        <h2>API posts</h2>
+        <Post />
+      </div>
+
+      <div>
+        <Link to={ACCOUNT}>Account page</Link>
```

Comment:
I don't know if this is good practice. For me, it didn't help much but make the "pathname" lookup more hard

Reply:
But it makes sense though, haha, just go with this.

Reply:
I think that we should do this, since we will link to the same page in multiple locations. Using such variables ensures that links don't break so easily.

Reply:
I see.